erdpy/cli_wallet.py

```diff
@@ -69,6 +87,29 @@ def setup_parser(args: List[str], subparsers: Any) -> Any:
     return subparsers
 
 
+def new_wallet(args: Any):
+    mnemonic = generate_mnemonic()
+    print(f"Mnemonic: {mnemonic}")
+    seed, pubkey = wallet.derive_keys(mnemonic)
```

Comment:
Warning: `wallet.derive_keys()` returns `private_key, public_key`, not a seed and a public key. Below, the function `save_to_key_file()` attempts to create the private key concatenating `seed + pubkey`, but `seed` already is the private key instead.